### ISSUE TITLE
Search highlight

### ISSUE THREAD

**Issue by lucabelmondo** [ygt-mikekchar/agnostic#11]
I've been trying a few combinations in both dark and light, to find something readable in both.

Current version: dark is OK, light is less readable.

![screen shot 2015-04-03 at 11 20 06](https://cloud.githubusercontent.com/assets/698828/6981069/f3b14b1a-d9f3-11e4-866b-163e058f5bee.png)   ![screen shot 2015-04-03 at 11 20 13](https://cloud.githubusercontent.com/assets/698828/6981070/f3b3246c-d9f3-11e4-82ac-5ce9bcbca524.png)

Some trials:

`highlight Search ctermbg=9 ctermfg=0`

![screen shot 2015-04-03 at 11 09 54](https://cloud.githubusercontent.com/assets/698828/6981120/919222dc-d9f4-11e4-99fd-52144190061e.png) ![screen shot 2015-04-03 at 11 10 14](https://cloud.githubusercontent.com/assets/698828/6981121/9192c85e-d9f4-11e4-98cb-b87481b4151d.png)

`highlight Search ctermbg=7 ctermfg=0`

![screen shot 2015-04-03 at 11 08 39](https://cloud.githubusercontent.com/assets/698828/6981131/b60757c2-d9f4-11e4-8467-c178191fb617.png) ![screen shot 2015-04-03 at 11 08 47](https://cloud.githubusercontent.com/assets/698828/6981132/b607b2e4-d9f4-11e4-821e-8e9bb88583ee.png)

`highlight Search ctermbg=3 ctermfg=0`

![screen shot 2015-04-03 at 11 06 57](https://cloud.githubusercontent.com/assets/698828/6981138/cafbe0b2-d9f4-11e4-8af4-7b7d183928a1.png) ![screen shot 2015-04-03 at 11 07 03](https://cloud.githubusercontent.com/assets/698828/6981139/cafc70cc-d9f4-11e4-946b-d0cc8071fe8a.png)

`highlight Search ctermbg=11 ctermfg=0`

![screen shot 2015-04-03 at 11 11 08](https://cloud.githubusercontent.com/assets/698828/6981103/64e98f54-d9f4-11e4-885b-c5340d7f596c.png) ![screen shot 2015-04-03 at 11 11 18](https://cloud.githubusercontent.com/assets/698828/6981104/64ea222a-d9f4-11e4-8134-8984212c8134.png)

My favourite is the last one, what do you think? Would you be happy for me to submit a pull request for it?

**Comment by mikekchar** [comment]
Yes.  Please send a PR.  I agree that it looks good.  I need to check it against the other supported palettes.
